<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addWindow</class>
 <widget class="QDialog" name="addWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adding a task</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_4"/>
     </item>
     <item row="7" column="1">
      <widget class="QDateEdit" name="dateEdit"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Surname</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Deadline</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Position</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Enter</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_3"/>
     </item>
     <item row="10" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>addWindow</receiver>
   <slot>result_saved()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>798</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>addWindow</receiver>
   <slot>to_close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>824</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>result_saved()</slot>
  <slot>to_close()</slot>
 </slots>
</ui>
